src/stylesheet.c

```diff
@@ -1144,14 +1147,18 @@ css_error css__stylesheet_rule_destroy(css_stylesheet *sheet, css_rule *rule)
 	case CSS_RULE_CHARSET:
 	{
 		css_rule_charset *charset = (css_rule_charset *) rule;
-		lwc_string_unref(charset->encoding);
+
+		if (charset->encoding != NULL)
+			lwc_string_unref(charset->encoding);
```

Comment:
Is there a real problem here or was this just added to the commit? Is there somewhere that's making charset rules without encodings?